news/2020-06-19-Bootable-jar-Wildfly-20.adoc

```diff
@@ -0,0 +1,107 @@
+= A bootable JAR for WildFly 20
+jfdenise
+2020-06-19
+:awestruct-tags: [wildfly, galleon, bootable, jar, bootable-jar]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+
+== Introduction
+
+Now that WildFly supports link:https://projects.eclipse.org/projects/technology.microprofile[Eclipse Microprofile] this seems the appropriate time to introduce 
+a way to package WildFly application as bootable JAR (bootable JAR packaging is well suited for microservices to be run on cloud and bare metal).
```

Comment:
s/package WildFly/package a WildFly/g
s/as bootable/as a bootable/g

I think the last part in parens should be separate sentence not in parens.